lib/new_relic/local_environment.rb

```diff
@@ -138,6 +139,12 @@ def check_for_puma
       end
     end
 
+    def check_for_falcon
+      if defined?(::Falcon::Server) && NewRelic::LanguageSupport.object_space_usable?
+        @discovered_dispatcher = :falcon if find_class_in_object_space(::Falcon::Server)
+      end
```

Comment:
An optional way to avoid having the entire method be a single `if` statement.

```suggestion
      return unless defined?(::Falcon::Server) && NewRelic::LanguageSupport.object_space_usable?
      
      @discovered_dispatcher = :falcon if find_class_in_object_space(::Falcon::Server)
```

Reply:
I like this cleanup! All the different webserver patterns follow this check. I'll make a cleanup ticket and label 'good first issue' to remove these giant `if` blocks in a consistent way.

Reply:
New ticket! https://github.com/newrelic/newrelic-ruby-agent/issues/2388